src/Symfony/Component/Process/Tests/ProcessTest.php

```diff
@@ -1669,7 +1669,7 @@ public function testIgnoringSignal()
             $this->markTestSkipped('pnctl extension is required.');
         }
 
-        $process = $this->getProcess('sleep 10');
+        $process = $this->getProcess(['sleep', '10']);
```

Comment:
Do we need to change line 1688 as well?

Reply:
The test works without changing it so... no I guess